plugins/node/opentelemetry-instrumentation-aws-sdk/test/bedrock.test.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  getTestSpans,
+  registerInstrumentationTesting,
+} from '@opentelemetry/contrib-test-utils';
+import { AwsInstrumentation } from '../src';
+registerInstrumentationTesting(new AwsInstrumentation());
+
+import {
+  BedrockRuntimeClient,
+  ConverseCommand,
+  ConversationRole,
+} from '@aws-sdk/client-bedrock-runtime';
+import * as path from 'node:path';
```

Comment:
```suggestion
import * as path from 'path';
```

Support for the `node:` prefix for builtin modules was added in v14.18.0, but "engines" for this package is currently "14", so it could fail for the poor sap still using the ancient early v14 version.